mandatory/2-function-creation.js

```diff
@@ -15,26 +28,87 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) 
+
+{
+  let result; 
+
+  if( totalPrice > 200)
+  {
+    result = (totalPrice - (totalPrice*0.10));
+  }
+  else
+  {
+    result = (totalPrice - (totalPrice*0.05));
+  }
+  return result;
+}
+
+
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(limit) 
+{
```

Comment:
Just a comment on coding style. Generally we would keep this opening `{` on the line above after the function parameters, e.g.
`function printOddNumbers(limit) {`
You can install Prettier on VSCode and set up format on save so that you always have your code in a consistent style inline with conventions. I can help with this if you need help. Here is an article explaining how https://www.educative.io/answers/how-to-set-up-prettier-and-automatic-formatting-on-vs-code

Reply:
I thought it would be easier to read this way. Definitely will use Prettier to let my code be formatted as standard and to look much prettier ; ) Thanks